### PR TITLE
Translate escripts.md to Portuguese

### PR DESCRIPTION
**Pull request by ruan-brandao**

### CODE REVIEW

File: pt/lessons/advanced/escripts.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+---
+
+Para criar executáveis em Elixir nós utilizaremos escript. Escript produz um executável que pode rodar em qualquer sistema que tenha Erlang instalado.
+
+## Sumário
+- [Começando](#getting-started)
+- [Fazendo Parsing dos Argumentos](#parsing-args)
+- [Criando o Executável](#building)
+
+## <a name="getting-started"></a>Começando
```
Comment (pragmaticivan) [inline pt/lessons/advanced/escripts.md]: You should use anchors in Portuguese.

**Comment by pragmaticivan** [comment]
Solid! Just update the anchors to Portuguese and squash the commits! Thanks!

**Comment by ruan-brandao** [comment]
Anchors updated :smiley:

File: pt/lessons/advanced/escripts.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+category: advanced
+order: 3
+lang: pt
+---
+
+Para criar executáveis em Elixir nós utilizaremos escript. Escript produz um executável que pode rodar em qualquer sistema que tenha Erlang instalado.
+
+## Sumário
+- [Começando](#comecando)
+- [Fazendo Parsing dos Argumentos](#fazendo-parsing-dos-argumentos)
```
Comment (pragmaticivan) [inline pt/lessons/advanced/escripts.md]: Sorry, I forgot about this one. If the title doesn't have accentuation, you don't need to create an anchor for that in the title.

Comment (ruan-brandao) [inline pt/lessons/advanced/escripts.md]: Ohh, thanks for the heads up. I'll update this one.

**Comment by pragmaticivan** [comment]
Solid @ruan-brandao can you squash the commits? Let me know if you need help with that.

**Comment by ruan-brandao** [comment]
I can use some help @pragmaticivan

**Comment by pragmaticivan** [comment]
cc @ruan-brandao  https://medium.com/p/forking-workflow-contribuindo-em-projetos-open-source-utilizando-git-parte-2-a1849204d02e I wrote this post :) By the end of the post you can see the squash part :) enjoy!

**Comment by ruan-brandao** [comment]
Thank you @pragmaticivan  :smile:

**Comment by pragmaticivan** [comment]
Thanks @ruan-brandao good to go.

**Pull request merged**